package.json

```
@@ -96,6 +96,8 @@
     "build:js": "rollup -c",
     "clean": "rm -rf dist tsconfig.tsbuildinfo",
     "dev": "run-p watch test:server",
+    "format": "prettier \"**/*.{mjs,js,json,ts,css,html,md,yml,yaml}\" --write --ignore-path .gitignore",
```

```suggestion
    "format": "prettier \"**/*.{cjs,js,json,md,ts}\" --write --ignore-path .gitignore",
```

I'd rather stick to the file types that this repo is actually using rather than a generic list. E.g. this list didn't include `cjs`, which is in the repo, but it included a bunch of file types that are not.